metadata-db-app/int_test/cmr/metadata_db/int_test/utility.clj

```diff
@@ -114,34 +107,39 @@
     [{"type" "trim_whitespace", "field" "platform", "order" -100},
      {"type" "priority", "field" "platform", "source_value" "Aqua", "order" 10, "priority" 10}]))
 
+(def service-json
```

Comment:
This should be the final UMM-S model.

Reply:
Right; which isn't ready yet ...

Reply:
I forgot to add 😄 . It was a joke that it would be really great if there were only 2 fields in the model.